docs/self-hosted/deployment/requirements.md

```diff
@@ -0,0 +1,128 @@
+
+If you want to be flexible about growing your hard drive space in the future consider mounting it using [logical volume management (LVM)](https://en.wikipedia.org/wiki/Logical_volume_management) so you can add more hard drives when you need them.
+
+Apart from a local hard drive you can also consider using a storage options provided by AWS, Azure, google cloud etc.
+
+### CPU
+
+CPU requirements are dependent on the usage and expected workload. Your exact needs may be more, depending on your workload. Your workload is influenced by factors such as - but not limited to - how active your users are, how many conversations do you recieve and the integrations which you are using.
+
+The following is the recommended minimum CPU hardware guidance for a handful of example Chatwoot conversation base sizes.
+
+- **4 cores** is the **recommended** minimum number of cores and supports up to 10000 conversations a day
+- 8 cores supports up to 20000 conversaations
+- More conversations? Consider scaling horizontally by adding more application servers
+
+### Memory
+
+Memory requirements are dependent on the usage and expected workload. Your exact needs may be more, depending on your workload. Your workload is influenced by factors such as - but not limited to - how active your users are, how many conversations do you recieve and the integrations which you are using.
+
+The following is the recommended minimum Memory hardware guidance for a handful of example Chatwoot conversation base sizes.
+
+- **4GB RAM** is the **required** minimum memory size and supports up to 10000 conversations a day
+  - we are always working to reduce the memory requirement.
+- 8GB RAM supports up to 20000 conversations
+- More conversations? Consider scaling horizontally by adding more application servers
+
+In addition to the above, we generally recommend having at least 2GB of swap on your server,
+even if you currently have enough available RAM. Having swap will help reduce the chance of errors occurring
+if your available memory changes. We also recommend configuring the kernel's swappiness setting
+to a low value like `10` to make the most of your RAM while still having the swap
+available when needed.
+
+## Database
+
+PostgreSQL is the only supported database. We don't have plans to support any other alternatives as of now.
+
+### PostgreSQL Requirements
+
+The server running PostgreSQL should have _at least_ 5-10 GB of storage
+available, though the exact requirements depends on the usage on your Chatwoot Instance.
+
+We highly recommend using the latest stable PostgreSQL versions as these were used for development and testing.
+
+### Redis 
+
+Redis stores the background task queue and various chatwoot configurations cached.
+The storage requirements for Redis are minimal, You can start with 100MB and scale up as required.
+
+Chatwoot requires Redis version 4.0 or higher.
+Redis version 6.0 or higher is recommended
```

Comment:
There is something missing here. I couldn't understand whether 4.0 or 6.0 is required.